### DISCUSSION TITLE
NocoDB themes [Jun 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2595] [Discord Threads]
> 🙋‍♂️ **chingun**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Open any project, open right top menu (3 dots) & select `Themes`  
![image0](https://cdn.discordapp.com/attachments/991213747362332842/991214969741922394/Screenshot_2022-06-28_at_11.02.03_AM.png)  
#  
> 🙋‍♂️ **chingun**  
#### In the code?  
#  
> 👨‍🦲 **dstala**  
#### <@404589929075834892> theme modifications are not supported via code / api currently  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/991213747362332842)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by spoorun** [reply]
Hi @o1lab are themes still available at the moment - with the new UI? Can't seem to find them (using self-hosted docker version).
Thanks
